pkg/apis/cloudcredential/v1/types_credentialsrequest.go

```diff
@@ -53,6 +53,12 @@ type CredentialsRequestSpec struct {
 	// ProviderSpec contains the cloud provider specific credentials specification.
 	// +kubebuilder:pruning:PreserveUnknownFields
 	ProviderSpec *runtime.RawExtension `json:"providerSpec,omitempty"`
+
+	// ServiceAccountNames contains a list of service accounts that will use permissions granted by this
+	// CredentialsRequest. This is not used by CCO but instead needed for ccoctl tool to set up access
+	// control in the cloud provider.
+	// +optional
+	ServiceAccountNames []string `json:"serviceAccountNames,omitempty"`
```

Comment:
Do we want `ServiceAccountNames` or just `ServiceAccounts`? My gut tells me `ServiceAccounts`, but the Pod spec does have fields for both `ServiceAccount` and `ServiceAccountName`.
WDYT?

Reply:
I would have gone with `ServiceAccounts` if it was a list of structs. But we are only storing names here (which is a specific attriibute of ServiceAccount) so `ServiceAccountNames` sounds more appropriate to me.

Reply:
SGTM